.metadata/RAG-prebuilt-integrations/setup-instructions/gdrive-pinecone-instructions.md

```
@@ -0,0 +1,72 @@
+## What it does
+- This RAG ingestion integration automatically ingests files from Google Drive and generates embeddings, storing them in your Pinecone vector database. These embeddings enable semantic search and retrieval for your RAG applications.
+
+<details>
+
+<summary>Data Source Setup</summary>
+
+#### Google Drive
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)